test_hello_worlds/hello.cpp

```diff
@@ -0,0 +1,6 @@
+#include <iostream>
```

Comment:
This file contains at least one console log. Please remove any present. - additions.length